http/handler_request_id.go

```diff
@@ -0,0 +1,22 @@
+package http
```

Comment:
The filename format is inconsistent for this file compared with the others - is it worth aligning it with the others?

Reply:
Should this code live with the identity.go file?

That way this file can be removed and the tests can also be moved to identity_test.go

Reply:
I moved this file (and its test) from handlers to http because the server, in http, was using it. If we had it in handlers then there would be an import cycle, because handlers import http.

I tried to flatten the handlers into http, but they rely on a tree structure of packages, with duplicated names of functions in different sibling packages, so at the end I just moved this file, which was needed.

But actually I can move this handler to identity.go. I agree that it makes sense.

About the other handlers, maybe we could refactor them in the future, flattening the structure to http package? But we might need to re-think some of the function names, as they would be duplicate and would be too long if we differenciate them by adding a prefix 'handleXYZ'

Reply:
Update: After @nshumoogum comments in the handlers package, I refactored them and now they have a flat structure. If we require to move them in the http package it should be easy now.